module-bsp/board/rt1051/bsp/magnetometer/magnetometer.cpp

```diff
@@ -294,37 +296,55 @@ namespace bsp
             return read == 1;
         }
 
+        bool sliderChangedPosition(const Measurements &current)
+        {
+            /// The magnetometer is quite noisy. In some cases, noise can switch slider mode.
+            /// So we did a few measurements to calculate the level of noises and it should work fine.
+            constexpr auto maxNoiseLevel = 45;
+            bool result                  = false;
+
+            // We don't check Z axis, because it isn't logic dependend
+            if ((std::abs(current.X - last.X) > maxNoiseLevel) || (std::abs(current.Y - last.Y) > maxNoiseLevel)) {
+                // LOG_INFO("\n**** MAG Delta : X:%d Y:%d\n", std::abs(current.X - last.X), std::abs(current.Y -
+                // last.Y));
```

Comment:
please remove logs, or incorporate them into code ;)

Reply:
I'll get rid of it later, it helps to test some code changes :)